lib/puppetx/puppetlabs/strings/yard/templates/default/html_helper.rb

```diff
@@ -0,0 +1,57 @@
+# A class containing helper methods to aid the generation of HTML
+# given formatted data
+class HTMLHelper
+
+  # Generates the HTML to format the relevant data about return values
+  def generate_return_types(types, desc = nil)
+    result = []
+
+    result << "(<span class=\"type\"><tt>" << types.join(", ") << "</tt></span>)"
+
+    if !desc.nil?
+      result << "- <div class=\"inline\"><p>#{desc}</p></div>"
+    end
+
+    result.join
+  end
+
+  # Generates the HTML to format the relevant data about parameters
+  def generate_parameters(params)
+    result = []
+
+    params.each do |param|
+      result << "<li>"
+
+      if !param[:exists?]
+        result << "<strike>"
+      end
+
+      result << "<span class=\"name\">#{param[:name]} </span>"
+      result << "<span class=\"type\">"
+
+      if param[:types]
+        result << "(" << "<tt>" << param[:types].join(", ") << "</tt>" << ")"
+      else
+        result << "(<tt>TBD</tt>)"
```

Comment:
When we discussed how to deal with parameters we decided that if the user did not specify a type we would list it as "TBD" for puppet manifests. I decided to keep it consistent for functions as well, but I'm happy to change it if you think something else would be more appropriate